web/cobrands/hart/_colours.scss

```diff
@@ -11,21 +11,21 @@ $primary_text: #ffffff;
 $primary_link_color: $primary_text;
 $primary_link_hover_color: rgba($primary_text, 0.8);
 
-$link-color: #369;
-$link-hover-color: #369;
+$link-color: #0077cc;
+$link-hover-color: #003078;
 
 $base_bg: #ffffff;
 $base_fg: #1a1a1a;
 
-/* Unused here */
-$nav_colour: #fff;
-$nav_hover_background_colour: #444;
+$nav_colour: $link-color;
+$nav_background_colour: #fff;
+$nav_hover_background_colour: #fff;
 
 $nav_background_colour: $primary;
-$mappage-header-height: 173px + 32px;
+$mappage-header-height: 205px;
 $header-top-border: false;
 
-$container-max-width: 60em;
+$container-max-width: 80em;
```

Comment:
This change doesn't seem to match the rest of the changes (not about tidying or colour changes, I mean). Just to match their site more?

Reply:
Oops, yes, sorry. Just to make it a bit less jarring, if you follow a link between the two sites.